README.md

```diff
@@ -51,10 +51,24 @@ $config = new Gr4vy\Gr4vyConfig("[YOUR_GR4VY_ID]", $privateKeyLocation, false, "
 ## Gr4vy Embed
 
 To create a token for Gr4vy Embed, call the `config->getEmbedToken()` function
-with the amount, currency, and optional buyer information for Gr4vy Embed.
+with the amount, currency, optional buyer information and optional checkout session for Gr4vy Embed.
 
 ```php
+//A checkout session allows multiple transaction attempts to be tied together
+$checkoutSession = $config->newCheckoutSession();
+
 echo $config->getEmbedToken(
+  array(
+    "amount"=> 200,
+    "currency" => "USD",
+    "buyer_id"=> "d757c76a-cbd7-4b56-95a3-40125b51b29c", 
+    "checkoutSessionId"=> $checkoutSession["id"]
+  )
+);
```

Comment:
This isn't right. The checkout session ID is stored on the root of the claims, not in the Embed data object.

Reply:
fixed